packages/datocms-plugin-table/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "datocms-plugin-complex-table-editor",
```

Comment:
@pbortnick should this be `@hashicorp/datocms-plugin-complex-table-editor`?

Reply:
It has to be datocms-plugin-xxx https://www.datocms.com/docs/plugin-sdk/publishing-to-marketplace
<img width="694" alt="Screen Shot 2022-08-15 at 12 15 34 PM" src="https://user-images.githubusercontent.com/20215887/184673500-cebfa80c-ce65-4d16-ad37-a42a34540d2b.png">

Reply:
@BRKalow ☝🏼 is there any issue in publishing without the `@hashicorp/` prefix with our npm publishing workflows?

Reply:
I think it should be okay! Let's give it a shot.

Reply:
If there is no changeset will anything be affected?

Reply:
@pbortnick Nope, without a changeset nothing will happen or be staged for publishing.